cmake/SSGCommon.cmake

```diff
@@ -914,6 +914,20 @@ macro(ssg_build_product PRODUCT)
             "
         )
     endif()
+    if(SSG_ANSIBLE_PLAYBOOKS_PER_RULE_ENABLED)
+        install(
+            CODE "
+            file(GLOB PLAYBOOK_PER_RULE_FILES \"${CMAKE_BINARY_DIR}/${PRODUCT}/playbooks/*\") \n
+            if(NOT IS_ABSOLUTE ${SSG_ANSIBLE_ROLE_INSTALL_DIR}/playbooks)
+                file(INSTALL DESTINATION \"\${CMAKE_INSTALL_PREFIX}/${SSG_ANSIBLE_ROLE_INSTALL_DIR}/playbooks/${PRODUCT}\"
```

Comment:
I'm sorry, I had a comment but I apparently didn't submit it or I misclicked. My comment was that the ${SSG_ANSIBLE_ROLE_INSTALL_DIR} is already used to install profile playbooks. But the downstream Linux distributions will want to place the rule playbooks to a different subpackage than the profile playbooks. Therefore I think it might be better to put the rule playbooks to a different directory than ${SSG_ANSIBLE_ROLE_INSTALL_DIR}. What is your opinion on that?

Reply:
I understand your concern, but it would feel really strange to have ansible playbooks on a different folder than `{SSG_ANSIBLE_ROLE_INSTALL_DIR}` (which is usually `scap-security-guide/ansible`). That's why a created a subfolder called `playbooks` where this files would be installed. I guess the fact that downstream Linux distributions will use these files on a different subpackage doesn't necessarily mean we have to install these files on a different top-level directory.

Reply:
>  but it would feel really strange to have ansible playbooks on a different folder than {SSG_ANSIBLE_ROLE_INSTALL_DIR}

It depends, something like /usr/share/scap-security-guide/rule_ansible_playbooks/ could work.

>  I guess the fact that downstream Linux distributions will use these files on a different subpackage doesn't necessarily mean we have to install these files on a different top-level directory.

That's correct, it's certainly possible to pick the right items for the (sub)packages in spec file.

I mean I think it's fine in the current form, I was curious if we could come up with something possibly better or not.

Reply:
As the profile playbooks installed in `${SSG_ANSIBLE_ROLE_INSTALL_DIR}/` are actually playbooks (not roles), how about installing the rule playbooks in `${SSG_ANSIBLE_ROLE_INSTALL_DIR}/rule_playbooks/${PRODUCT}`?

Reply:
I agree that the naming is a bit confusing here. I guess it's better to have explicit than some hidden meaning. I'll update it accordingly. Thanks